app/dashboard/templates/shared/profile_activities_bounty.html

```diff
@@ -1,45 +1,64 @@
-<a class="row result bounty_row {{ bounty.network }}" target="_blank"
- {% if bounty.url %} href="{{ bounty.url }}"
- {% elif bounty.github_url %} href="{{ bounty.github_url }}"
+{% load i18n humanize %}
+<a class="row result bounty_row {{ activity.bounty.network }}" target="_blank"
+ {% if activity.bounty.url %} href="{{ activity.bounty.url }}"
+ {% elif activity.bounty.github_url %} href="{{ activity.bounty.github_url }}"
  {% endif %}>
   <div class="avatar-container col-1 justify-content-center special_tag hide_min_viewport">
-    <img class="avatar" src="{% url 'org_avatar' bounty.bounty_owner_github_username %}?repo=https://github.com/{{ bounty.bounty_owner_github_username }}">
+    <img class="avatar" src="{% url 'org_avatar' activity.bounty.bounty_owner_github_username %}?repo=https://github.com/{{ activity.bounty.bounty_owner_github_username }}">
   </div>
   <div class="col-12 col-md-8 d-flex flex-column">
     <div class="bounty-detail">
       <div class="title font-subheader">
-        {{ bounty.title }}
+        {{ activity.bounty.title }}
       </div>
       <div class="bounty-summary col-12">
         <div class="info font-caption">
-          {% if bounty.experience_level %}
-            {{ bounty.experience_level }} &bull;
+          {% if activity.bounty.experience_level %}
+            {{ activity.bounty.experience_level }} &bull;
           {% endif %}
-          {% if bounty.status %}
-            {{ bounty.status|title }}
+          {% if activity.bounty.status %}
+            {{ activity.bounty.status|title }}
           {% endif %}
+          {% if activity.activity_type == 'new_bounty' %}
```

Comment:
I'm still interested to see how this performs on live with fairly populated activities 🤔I'm fine with it, but we might need to patch it depending on the performance.

Reply:
Yes. I agree to that. I would like to see the same.